lib/sla.js

```diff
@@ -40,54 +41,108 @@ function designateMixingBridge(client, bridgeName) {
 }
 
 /**
- * Adds inbound and originated channels to the bridge.
- * @param {Object} inbound - Incoming channel to be added to the bridge.
+ * Updates the state of the shared extension.
+ * @param {Object} client - Object that contains information from the ARI
+ *   connection.
+ * @param {String} name - name of the shared extension to update.
+ * @param {String} state - state to update the shared extension to.
+ */
+function updateState(client, name, state) {
+  var deviceState = Q.denodeify(client.deviceStates.update.bind(client));
+  deviceState({deviceName: 'Stasis:' + name, deviceState: state});
```

Comment:
Only problem here is that since you don't return deviceState() there's no way for the caller to determine if an error occurred.